contrib/julia-config.jl

```diff
@@ -8,6 +8,14 @@ const options =
     "--ldlibs"
 ];
 
+function threadingOn()
+    try
+        return cglobal(:jl_tls_states) == C_NULL
+    catch
+        return true
```

Comment:
that's not quite what I would have naively expected. comment that says as much maybe?